backend/grant/templates/emails/template.txt

```diff
@@ -3,7 +3,8 @@
 ===============
 
 {{ UI.NAME }}
```

Comment:
`{{ UI.NAME }}` should be removed here. It is set to "SF Grants" in `settings.py`

Reply:
Cool, I'll get this removed.